entities/JsonHandlers/Serializers/EntityJsonSerializer.cpp

```diff
@@ -63,32 +56,28 @@ EntityJsonSerializer::toJson(Way *entity, rapidjson::Document::AllocatorType &al
 
     json = setStringVector(std::move(json), "restricted", entity->getRestricted(), allocator);
 
-    rapidjson::Value curves;
-    curves.SetArray();
-    for (Coordinates coord : entity->getCurves()) {
-        rapidjson::Value coordVal = coordinatesJsonSerializer.toJson(coord, allocator);
-        curves.PushBack(coordVal, allocator);
-    }
+    Geometry *geometry = entity->getGeometry().get();
```

Comment:
Same comment as before:
It's generally better to not use get().
I used it elsewhere because I needed to send objects as function parameters and use overloading.